pkg/karmadactl/promote.go

```diff
@@ -68,27 +68,23 @@ func NewCmdPromote(cmdOut io.Writer, karmadaConfig KarmadaConfig, parentCommand
 	return cmd
 }
 
-func getPromoteExample(parentCommand string) string {
-	promoteExample := `
-# Promote deployment(default/nginx) from cluster1 to Karmada.
-` + parentCommand + ` promote deployment nginx -n default -c cluster1
+func promoteExample(parentCommand string) string {
+	example := `
+# Promote deployment(default/nginx) from cluster1 to Karmada` + "\n" +
+		fmt.Sprintf("%s promote deployment nginx -n default -c cluster1", parentCommand) + `
 
-# Promote deployment((default/nginx) with gvk from cluster1 to Karmada.
-` + parentCommand + ` promote deployment.v1.apps nginx -n default -c cluster1
+# Promote deployment((default/nginx) with gvk from cluster1 to Karmada` + "\n" +
+		fmt.Sprintf("%s promote deployment.v1.apps nginx -n default -c cluster1", parentCommand) + `
 
-# Dumps the artifacts but does not deploy them to Karmada, same as 'dry run'.
-` + parentCommand + ` promote deployment nginx -n default -c cluster1 -o yaml|json
+# Dumps the artifacts but does not deploy them to Karmada, same as 'dry run'` + "\n" +
+		fmt.Sprintf("%s promote deployment nginx -n default -c cluster1 -o yaml|json", parentCommand) + `
 
-# Promote secret(default/default-token) from cluster1 to Karmada.
-` + parentCommand + ` promote secret default-token -n default -c cluster1
-
-# For clusters with 'Pull' mode, use '--cluster-kubeconfig' to specify the configuration.
-` + parentCommand + ` promote deployment nginx -n default -c cluster1 --cluster-kubeconfig=<CLUSTER_KUBECONFIG_PATH>
-
-# For clusters with 'Pull' mode, use '--cluster-kubeconfig' and '--cluster-context' to specify the configuration.
-` + parentCommand + ` promote deployment nginx -n default -c cluster1 --cluster-kubeconfig=<CLUSTER_KUBECONFIG_PATH> --cluster-context=<CLUSTER_CONTEXT>
```

Comment:
Why delete this example?

Reply:
It feels too long. BTW, I will remove these parameters `--cluster-kubeconfig` and `--cluster-context` and let `promote` command to use AA in next pr.

Reply:
> BTW, I will remove these parameters --cluster-kubeconfig and --cluster-context and let promote command to use AA in next pr.

Why? We can't restrict users use AA, isn't it? Mybe the `--cluster-kubeconfig` and `--cluster-context` is a more straightforward way.

Reply:
> Why? We can't restrict users use AA, isn't it? Mybe the --cluster-kubeconfig and --cluster-context is a more straightforward way.

If uniform authorization is to be implemented, we can consider uniformly using AA to restrict access to member clusters. 

It is better to use AA to unify the PUSH and PULL modes, otherwise users must provide `kubeconfig` to operate the PULL mode cluster, which is cumbersome to use.

If we consider that the user does not use AA, we can have `promote` support AA but retain `--cluster-kubeconfig` and `--cluster-context`.